### ISSUE TITLE
Forbid to use `continue` in `finally`

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1082]
# Rule request

## Thesis

This code is now allowed:

```python
for some in range(10):
    try:
        func(some)
    finally:
        continue
```

Why would you do that? 
Putting any control statements in `finally` is a terrible practice.
We should not allow it.

It was added in `python3.8`: https://www.python.org/downloads/release/python-381/

**Comment by Kvm99** [comment]
Hi, could I take the issue?

**Comment by sobolevn** [comment]
Sure! Feel free to ask for help.

**Comment by Kvm99** [comment]
Hi! I am trying to run and write tests and have an error: 
```ModuleNotFoundError: No module named 'plugins' ```
I could not find any information about such module in the internet and have no idea how to deal with the problem.
Could you help me, please?

**Comment by sobolevn** [comment]
Sure! It looks like a problem with your environment.

Can you please post how exactly you run your tests?
You can also take a look at how we run our tests here: https://github.com/wemake-services/wemake-python-styleguide/blob/master/.travis.yml#L19-L20

**Comment by Kvm99** [comment]
I tried to run tests from the project root by ```make test``` and ```pytest```. Each time there was the same error: ```ModuleNotFoundError: No module named 'plugins' ```
Then I changed ```plugins.violation``` to ```test.plugins.violation``` in conftest.py and it helped.

**Comment by sobolevn** [comment]
Two questions:
1. How does your installation process looks like? Because it is proven to work on CI and locally
2. What is your OS?

**Comment by Kvm99** [comment]
I was going the following way:
```
pyenv update
pyenv local 3.6.7
pip3 install poetry
poetry install
poetry shell
make test
```

**Comment by sobolevn** [comment]
Looks strange. This should do the trick.

Tow comments, however:
1. Use should use `3.8.0`, because this feature is only possible there
2. You can try to `activate` env instead of using `poetry shell`. Try `source $(poetry env info -p)/bin/activate`

**Comment by sobolevn** [comment]
@Kvm99 can you please also test `break` keyword in `finally`? Is it allowed? If so, we need to forbid it as well.

**Comment by sobolevn** [comment]
Please, also specify that this rule is related to `TryExceptMultipleReturnPathViolation`

**Comment by Kvm99** [comment]
> @Kvm99 can you please also test `break` keyword in `finally`? Is it allowed? If so, we need to forbid it as well.

I checked. `break` in `finally` works with python 3.6, 3.7, 3.8. 
Should not we add a new issue for such case? I will also do it.

**Comment by sobolevn** [comment]
We can combine them into one: `continue` and `break`
